CHANGELOG.md

```diff
@@ -2,9 +2,12 @@
 All notable changes to this project will be documented in this file.
 This project adheres to Semantic Versioning (http://semver.org/).
     
-## 3.11.1
-- Fix Jenkins checkstyle reports
-- Rewrite Jenkinsfile to use modern format 
+## 3.12.0
+* Fix Jenkins checkstyle reports
+* Rewrite Jenkinsfile to use modern format 
+* Add `APPS` token as a default token to be generated on account creation
+* Add helper method to check array contains only scalar values
+* Refactor method name in HtmlMarkupSerializationHelper
```

Comment:
Which method? How about just: Rename `HtmlMarkupSerializationHelper::canEndoded` to `HtmlMarkupSerializationHelper::canBeEncoded`